bgl/mcgregor.py

```diff
@@ -226,14 +260,20 @@ def recursion(
         return
 
     num_edges = arcs_left(marcs)
-
-    if num_edges < threshold:
+    num_edges_2 = arcs_left(transpose_marcs(marcs, g2.n_edges))
```

Comment:
Does breaking this up into two statements make a performance difference?

Reply:
reverting this back to array of uint8